app/src/pages/inside/dashboardItemPage/modals/common/widgetControls/controls/filtersControl/filtersControl.jsx

```diff
@@ -184,6 +199,30 @@ export class FiltersControl extends Component {
     conditions: filter.conditions.filter((item) => item.value.trim()),
   });
 
+  getFiltersList = (filters) => {
+    if (!filters.length) return [];
+
+    const { activeLaunchFilter } = this.props;
+
+    if (!activeLaunchFilter) return filters;
+
+    const filtersList = [...filters].filter((el) => el.id !== activeLaunchFilter.id);
```

Comment:
Unnecessary destructurization.  `Array.filter` always return the new array of filtered items.